modules/ROOT/pages/sync/binding_to_keycloak.adoc

```diff
@@ -0,0 +1,30 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Binding the {sync-service} service to a Keycloak instance
+
+This section will help you to bind a Keycloak service to the {sync-service} service.
+
+== Prerequisites
+
+* Ensure {sync-service} is xref:sync/provisioning.adoc[provisioned].
+* Ensure {keycloak-service} is xref:keycloak/provisioning.adoc[provisioned]
+
+== Creating the binding secret from a Keycloak instance
+
+* Navigate to your Openshift project
+* Select *Applications* and then *Provisioned Services*
+* Select the *Identity Management* service from the list of Provisioned Services
+* Select the *Create Binding* option
+* Enter the *Service Instance Id* of the provisioned *{sync-service}* service
```

Comment:
Putting "newbie" hat > What is service instance ID?
 Maybe we can just say: Go to sync service section. Copy the Id.. etc. would it be possible?